staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go

```diff
@@ -2706,7 +2732,32 @@ var podColumns = []metav1.TableColumnDefinition{
 
 // build a meta table response from a pod list
 func podTableObjBody(codec runtime.Codec, pods ...corev1.Pod) io.ReadCloser {
+	table := &metav1beta1.Table{
+		TypeMeta:          metav1.TypeMeta{APIVersion: "meta.k8s.io/v1beta1", Kind: "Table"},
+		ColumnDefinitions: podColumns,
+	}
+	for i := range pods {
+		b := bytes.NewBuffer(nil)
+		codec.Encode(&pods[i], b)
+		table.Rows = append(table.Rows, metav1beta1.TableRow{
+			Object: runtime.RawExtension{Raw: b.Bytes()},
+			Cells:  []interface{}{pods[i].Name, "0/0", "", int64(0), "<unknown>", "<none>", "<none>", "<none>", "<none>"},
+		})
+	}
+	data, err := json.Marshal(table)
+	if err != nil {
+		panic(err)
+	}
+	if !strings.Contains(string(data), `"meta.k8s.io/v1beta1"`) {
+		panic("expected v1beta1, got " + string(data))
+	}
```

Comment:
this ensures we're serializing the data we expected (the type aliases were throwing off the codec's preferred external version)

Reply:
Ah